core/src/main/java/org/kiwix/kiwixmobile/core/main/CoreMainActivity.java

```diff
@@ -460,8 +463,10 @@ private void showTabSwitcher() {
     actionBar.setDisplayHomeAsUpEnabled(true);
     actionBar.setHomeAsUpIndicator(
       ContextCompat.getDrawable(this, R.drawable.ic_round_add_white_36dp));
+    actionBar.setHomeActionContentDescription(
+      getResources().getString(R.string.open_new_tab)
```

Comment:
You don't need to call `getResources()`, just `getString()` will do the job but also there is a version of `setHomeActionContentDescription` that takes an int so you can just say
`actionBar.setHomeActionContentDescription(R.string.open_new_tab)`